docs/central-entities.rst

```diff
@@ -65,67 +75,64 @@ In this Quick Start, you will upload two Forms we have already authored for you:
 First, let's prepare the Forms for use with Central.
 
 1. Create a new Project, if you can. It can be easier to see what's going on without other Forms in the way.
-2. Upload the `Tree registration Form <https://docs.google.com/spreadsheets/d/1xboXBJhIUlhs0wlblCxcQ3DB5Ubpx2AxLDuaXh_JYyw/edit#gid=2050654322>`_. Notice how it has seen from the Form definition that the Form creates Entities in a Dataset, and populates three Entity Properties.
-3. Publish the Form. As the dialog says, this action creates the ``trees`` Dataset within the Project you created.
+2. Upload the `Tree registration Form <https://docs.google.com/spreadsheets/d/1xboXBJhIUlhs0wlblCxcQ3DB5Ubpx2AxLDuaXh_JYyw/edit#gid=2050654322>`_. Notice how Central knows from the Form definition that the Form creates Entities in an Entity List, and populates three Entity Properties.
```

Comment:
We should update "dataset" --> "list_name" in this demo form.

Reply:
Yes, yes, thank you! I'll set myself a reminder to do that when we merge.